apps/flame-defi/app/pool/types.ts

```diff
@@ -67,26 +69,70 @@ export type Positions = {
   [key: number]: Position;
 };
 
+export type PoolPositionResponse = {
+  nonce: bigint;
+  operator: string;
+  tokenAddress0: Address;
+  tokenAddress1: Address;
+  fee: number;
+  tickLower: number;
+  tickUpper: number;
+  liquidity: bigint;
+  feeGrowthInside0LastX128: bigint;
+  feeGrowthInside1LastX128: bigint;
+  tokensOwed0: bigint;
+  tokensOwed1: bigint;
+};
+
+export type GetAllPoolPositionsResponse = {
```

Comment:
You can still use `&`
```suggestion
export type GetAllPoolPositionsResponse = PoolPositionResponse & {
```